common/src/main/java/com/wynntils/models/containers/LootChestModel.java

```diff
@@ -61,11 +75,21 @@ public void onSetSlot(ContainerSetSlotEvent.Post event) {
         if (wynnItem.isEmpty()) return;
         GearBoxItem gearBox = wynnItem.get();
         if (gearBox.getGearTier() == GearTier.MYTHIC) {
+            WynntilsMod.postEvent(new MythicFoundEvent(itemStack));
             if (gearBox.getGearType() != GearType.MASTERY_TOME) {
+                mythicFinds
+                        .get()
+                        .add(new MythicFind(
```

Comment:
While you're at it, why not also store the current timestamp, and the players coordinates?